sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -184,6 +184,17 @@ def test_int_overflow_mutual_info_score():
     assert_all_finite(mutual_info_score(x.ravel(), y.ravel()))
 
 
+def test_int_overflow_fowlkes_mallows_score():
+    # cribbed directly from test_int_overflow_mutual_info_score
```

Comment:
Well maybe we should use a loop instead.

Reply:
What about just merging them into the same function, say `test_int_overflow_mutual_info_fowlkes_mallows_score`?

Reply:
Sounds reasonable